gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
```

Comment:
If we're going past 7.0, we might as well go to 7.1.1 or above. But we should probably stick with 6.9 considering the new requirements.

Reply:
i agree, might as well go with the latest 7.x